addons/website/static/src/components/views/page_views_mixin.js

```diff
@@ -86,8 +87,9 @@ export const PageRendererMixin = (component) => class extends component {
      *        specific clones).
      */
     recordFilter(record, records) {
+        const firstWebsiteId = record.data.website_id && record.data.website_id[0];
```

Comment:
`firstWebsiteId`? Isn't this juste the website_id of the current record we are processing in the loop? What do you mean by `first`?
`record_website_id` or juste `website_id` if that's the case IMO